content/blog/tidymodels-survival-analysis/index.Rmd

```diff
@@ -0,0 +1,129 @@
+
+
+## Ch-ch-changes: What's new for censored regression?
+
+The main components needed for this full-fledged integration of survival analysis into tidymodels were
+
+- Survival analysis models that can take censoring into account
+- Survival analysis performance metrics that can take censoring into account
+- Integrating changes required by these models and metrics into the framework
+
+For the models, parsnip gained a new mode, `"censored regression"`, for existing models as well as new model types such as `proportional_hazards()`. Engines for these reside in censored, the parsnip extension package for survival models. The `"censored regression"` mode has been around for a while and we've previously shared posts on [our initial thoughts](https://www.tidyverse.org/blog/2021/11/survival-analysis-parsnip-adjacent/) and the [release of censored](https://www.tidyverse.org/blog/2022/08/censored-0-1-0/).
+
+Now we've added the metrics: yardstick v1.3.0 includes new metrics for assessing censored regression models. Somewhat similar to how metrics for classification models can take class predictions or probability predictions as input, these survival metrics can take predicted survival times or predictions of survival probabilities as input. 
+
+The new metrics are
+
+- Concordance index on the survival time via `concordance_survival()`
+- Brier score on the survival probability and its integrated version via `brier_survival()` and `brier_survival_integrated()`
+- Area under the ROC curve on the survival probabilities via `auc_roc_survival()`
+
+The probability of survival is always defined _at a certain point in time_. We call that time point the evaluation time because it is then also the time point at which we want to evaluate model performance. Metrics that work on the survival probabilities are also called _dynamic metrics_ and you can read more about them here:
+
+- [Dynamic Performance Metrics for Event Time Data](https://www.tidymodels.org/learn/statistics/survival-metrics/)
+- [Accounting for Censoring in Performance Metrics for Event Time Data](https://www.tidymodels.org/learn/statistics/survival-metrics-details/)
+
+The evaluation time is also the best example to illustrate the changes necessary to the framework. Most of them were under the hood but the evaluation time is user-facing. Let's take a look at that.
```

Comment:
The next few paragraphs feel a little meandering. Can we make the last sentence

> Let's take a look at that:

and make the next few paragraphs bullets? Maybe that will read more focused to me.

Reply:
I've turned the three places of where we need to specify `eval_time` into bullet points, I think that works well.